lib/widgets/course_detail_widgets.dart

```diff
@@ -82,10 +84,19 @@ class GroupDetails extends StatelessWidget {
       "Schedule:",
       style: themeData.textTheme.headline4,
     ));
-    if (_group.schedule != null) {
-      view.add(EventCalendar(_group.schedule!));
+    try {
+      view.add(EventCalendar(_group.getSchedule()));
+    } catch (e) {
+      // do nothing
     }
 
+    // Minimum requirements
+    view.add(Text(
```

Comment:
Let's add padding around these elements and increase font size!
I find it way too small to read and also very bunched up together, it looks too dense. 
Also, to reconsider is the font colour, I feel like it's a little bit too light.